core/src/io/anuke/mindustry/world/blocks/distribution/ArmoredConduit.java

```diff
@@ -0,0 +1,46 @@
+package io.anuke.mindustry.world.blocks.distribution;
+
+import io.anuke.arc.Core;
+import io.anuke.arc.graphics.g2d.Draw;
+import io.anuke.arc.graphics.g2d.TextureRegion;
+import io.anuke.mindustry.type.Liquid;
+import io.anuke.mindustry.world.Block;
+import io.anuke.mindustry.world.Edges;
+import io.anuke.mindustry.world.Tile;
+
+public class ArmoredConduit extends Conduit{
+
+    protected TextureRegion capRegion;
```

Comment:
This region is so subtle that I don't think most players will notice it. Perhaps don't draw it at all, or make it a bit more solid.

Reply:
removing it also removes the justification as to why it leaks less (the end would literally be open), and if i made it any larger it would get in the way of the directional blends, personally i think its just the right size, most would likely notice it. 🤔